api/main.js

```diff
@@ -14,6 +14,7 @@ const server = express();
 endpointCoverage.registerCoverageMiddleware(server);
 server.use((req, res, next) => {
   req.id = uuid();
+  req.meta = {};
```

Comment:
Since the new middlewares add stuff to `req.meta`, I wanted to make sure it was defined first.  That way each downstream middleware doesn't have to check for it and create it as necessary.